client/src/components/Sections/Features/Features.tsx

```diff
@@ -0,0 +1,258 @@
+          </Grid>
+        </Grid>
+      </FeatureItem>
+      <FeatureItem fromDirection="left">
+        <Typography color="textPrimary" variant="h6" sx={{ textAlign: 'left' }}>
+          <EditIcon />
+          {' '}
+          Content Management
+        </Typography>
+        <Grid container>
+          <Grid item xs={12} sm={12} md={8}>
+            <List>
+              <ListItem>
+                <ListItemIcon>
+                  <BulletIcon color="primary" />
+                </ListItemIcon>
+                <Typography color="textSecondary">
+                  Manage and create new topics, source code snippets, lessons and
+                  quizzes
+                </Typography>
+              </ListItem>
+            </List>
+            <List>
+              <ListItem>
+                <ListItemIcon>
+                  <BulletIcon color="primary" />
+                </ListItemIcon>
+                <Typography color="textSecondary">
+                  Community-contributed resources for all students
+                </Typography>
+              </ListItem>
+            </List>
+          </Grid>
+          <Grid
+            item
+            xs={12}
+            sm={12}
+            md={4}
+            sx={{ position: 'relative', height: '200px' }}
+          >
+            <LaptopFrame imageUrl={cmsScreen} />
+          </Grid>
+        </Grid>
+      </FeatureItem>
+      <FeatureItem fromDirection="right">
+        <Typography color="textPrimary" variant="h6">
+          <GradCapIcon />
+          {' '}
+          Open source
+        </Typography>
+        <Typography color="textSecondary" variant="body1">
+          Developed and maintained by passionate CS and engineering students at UNSW.
+        </Typography>
+        <List>
+          <ListItem>
+            <ListItemIcon>
+              <BulletIcon color="primary" />
+            </ListItemIcon>
+            <Typography color="textSecondary">
+              <Link
+                href="https://github.com/csesoc/Structs.sh"
+                color="textSecondary"
+                target="_blank"
+              >
+                GitHub
+              </Link>
+            </Typography>
+          </ListItem>
+          <ListItem>
+            <ListItemIcon>
+              <BulletIcon color="primary" />
+            </ListItemIcon>
+            <Typography color="textSecondary">
+              The Structs.sh
+              {' '}
+              <RouterLink to="/about">
+                <Link color="textSecondary">team</Link>
+              </RouterLink>
+            </Typography>
+          </ListItem>
+        </List>
+        <Box sx={{ mt: 4 }}>
+          <div
+            className="github-card"
+            data-github="csesoc/structs.sh"
+            data-width="100%"
+            data-theme="medium"
+          />
+        </Box>
+      </FeatureItem>
+    </motion.div>
+  );
+};
+
+export default Features;
```

Comment:
Could this be broken up into smaller components? Also, should we be hiding some of the 'features' that we don't have yet before this hits production?

Reply:
Yeah we could hide some stuff but i wonder what the best way to do that would be